README-main.md.gotmpl

```diff
@@ -1,61 +0,0 @@
-# Woodpecker Helm Charts
```

Comment:
Both files are used. The `README.md.gotmpl` is used for the sub-charts and this one for the main readme.